sdk/identity/identity/src/credentials/managedIdentityCredential/cloudShellMsi.ts

```diff
@@ -33,6 +34,9 @@ function prepareRequestOptions(
   if (clientId) {
     body.client_id = clientId;
   }
+  if (resourceId) {
+    body.mi_res_id = resourceId;
```

Comment:
Cloudshell also doesn't support this parameter.

Reply:
For now, I will disable these MSIs that don’t support resourceId. Meaning that, if `resourceId` is passed, these MSIs will log a message and they won’t be picked to get a token.